src/routes/authRoutes.ts

```diff
@@ -0,0 +1,12 @@
+import { Hono } from 'hono';
+import { authController } from '@/controllers/authController.ts';
+import { validateRegistration, validateSign } from '@/middlewares/authValidation';
+import { jwt } from 'hono/jwt';
+
+const secretKey = process.env.SECRET_KEY!;
+
+export const authRoutes = new Hono().basePath('/auth');
+
+authRoutes.get('/page', jwt({ secret: secretKey }), authController.getPayload);
+authRoutes.post('/sign', validateSign, authController.getToken);
```

Comment:
You don’t need such a middleware here, please replace with zValidator similar to UserBodySchema and move logic from the validateSign to authController.getToken